gctscript/vm/vm_test.go

```diff
@@ -0,0 +1,220 @@
+package vm
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/gofrs/uuid"
+)
+
+var (
+	testScript       = filepath.Join("..","..","testdata","gctscript","once.gct")
+	testInvalidScript       = filepath.Join("..","..","testdata","gctscript","broken.gct")
+	testScriptRunner = filepath.Join("..","..","testdata","gctscript","timer.gct")
+)
+
+func TestNewVM(t *testing.T) {
+	x := New()
+	xType := reflect.TypeOf(x).String()
+	if xType != "*vm.VM" {
+		t.Fatalf("vm.New should return pointer to VM instead received: %v", x)
+	}
+}
+
+func TestVMLoad(t *testing.T) {
+	GCTScriptConfig = configHelper(true, true, 0, 6)
+	testVM := New()
+	err := testVM.Load(testScript)
```

Comment:
Please also add a test with "once" to test the ability to handle both "once" and "once.gct"

Reply:
Added coverage for this :+1: